### PR TITLE
50.json

### PR DESCRIPTION
**Pull request by RickyVanz**
adding new bolt-action anti material rifle, Pindad SPR-2.

#### Summary
Content: "added Pindad SPR-2, Bolt-action, Anti Material Sniper rifle."
#### Purpose of change
adding new weapon + learning.

### CODE REVIEW

File: data/json/items/gun/50.json [hunk 1]
```diff
@@ -70,5 +70,29 @@
       "barrel_length": -1
     },
     "flags": [ "RELOAD_EJECT" ]
+  },
+  {
+    "id": "spr_2",
+    "copy-from": "rifle_base",
+    "type": "GUN",
+    "name": "Pindad SPR 2",
+    "name_plural": "Pindad SPR-2"
```
Comment (mlangsdorf) [inline data/json/items/gun/50.json]: Missing a comma here.

Comment (RickyVanz) [reply data/json/items/gun/50.json]: im sorry for the mistakes, im still learning the JSON codes based on available material on the list and try to create one based on the material, so should i fix it now?

**Comment by RickyVanz** [comment]
im sorry, im still learning in the JSON code thanks to the community who invites me to join in development.
so what should i do to make this item valid?

**Comment by Xion350** [comment]
This is probably better done as a mod instead of an addition to the base game.  The SPR is about the same as the Barrett so there's no real reason to include it other than to add variety.

**Comment by I-am-Erk** [comment]
Between this and the keris, I'd say you might be well suited to making an 'indonesian armoury' type mod, that introduces your favourite indonesian weapons. They don't really fit the base game, but some people do love the variety.

File: data/json/items/gun/50.json [hunk 2]
```diff
@@ -70,5 +70,29 @@
       "barrel_length": -1
     },
     "flags": [ "RELOAD_EJECT" ]
+  },
+  {
+    "id": "spr_2",
+    "copy-from": "rifle_base",
+    "type": "GUN",
+    "name": "Pindad SPR 2",
+    "name_plural": "Pindad SPR-2",
```
Comment (Night-Pryanik) [inline data/json/items/gun/50.json]: Singular name is `SPR 2`, while plural name is `SPR-2`. Which is right?

File: data/json/items/gun/50.json [hunk 3]
```diff
@@ -70,5 +70,29 @@
...
+    "description": "Bolt-action, Anti Material-rifle designed by Pindad Armaments, its medium size and weight is offset by its range and damage.",
```
Comment (Night-Pryanik) [inline data/json/items/gun/50.json]:
```suggestion
    "description": "Bolt-action anti-materiel rifle designed by Pindad Armaments, its medium size and weight is offset by its range and damage.",
```

File: data/json/items/gun/50.json [hunk 4]
```diff
@@ -70,5 +70,29 @@
...
+    "material": [ "steel", "wood" ],
+    "ammo": "50",
+    "range": 50,
+    "ranged_damage": -5,
+    "dispersion": 80,
+    "durability": 8,
+    "barrel_length": 5,
+    "default_mods": [ "bipod", "rifle_scope", "muzzle_brake" ],
+    "magazine_well": 1,
+    "magazine": [ [ "50", [ "spr2mag" ] ] ],
```
Comment (Night-Pryanik) [inline data/json/items/gun/50.json]: You have to add magazine `spr2mag` in game files too.

File: data/json/items/gun/50.json (hunk 3 above)
Comment (RickyVanz) [reply data/json/items/gun/50.json]: thanks, blame it to my auto-capital keyboard function, it is a typo-maker.

**Pull request closed without merging**